internal/compiler-interface/src/main/java/xsbti/compile/DefinesClass.java

```diff
@@ -1,12 +1,14 @@
 package xsbti.compile;
 
 /**
-* Determines if an entry on a classpath contains a class.
-*/
-public interface DefinesClass
-{
- 	/**
-	* Returns true if the classpath entry contains the requested class.
-	*/
-	boolean apply(String className);
+ * Determines if a classpath entry contains a class.
+ *
+ * The corresponding classpath entry is not exposed by this interface. It's tied
+ * to it by a specific class that implements this interface.
+ */
+public interface DefinesClass {
+    /**
+     * Returns true if the classpath entry contains the requested class.
+     */
+    boolean apply(String className);
```

Comment:
Since the calling side looks like this,

```scala
val entries = classpath.toStream.map { entry => (entry, lookup.definesClass(entry)) }
```

and if there's frequent enough call, it might make sense to avoid instantiation of `Maybe`s.
If we're going to change it, I would either rename `apply` to some verb phrase or use [`xsbti.F1`](https://github.com/sbt/util/blob/1.0/internal/util-interface/src/main/java/xsbti/F1.java).